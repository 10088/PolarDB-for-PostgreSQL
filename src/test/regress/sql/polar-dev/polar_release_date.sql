-- Test polar_release_date
-- 
-- we can not show, because different versions may have different results.
-- show polar_release_date;
set polar_release_date="20150101";
-- show polar_release_date;
reset all;
-- show polar_release_date;
reset polar_release_date;
-- show polar_release_date;
