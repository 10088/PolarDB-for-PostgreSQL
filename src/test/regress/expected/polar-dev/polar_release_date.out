-- Test polar_release_date
-- 
-- we can not show, because different versions may have different results.
-- show polar_release_date;
set polar_release_date="20150101";
ERROR:  parameter "polar_release_date" cannot be changed
-- show polar_release_date;
reset all;
-- show polar_release_date;
reset polar_release_date;
ERROR:  parameter "polar_release_date" cannot be changed
-- show polar_release_date;
